app/src/Components/Project/ProjectFullDetails.tsx

```diff
@@ -63,6 +67,11 @@ const ProjectRelatedRoles = styled.Text`
   border: ${props => `2px solid ${props.theme.colors.staBlack}`};
   padding: 17px 27px 11px 27px;
 `
+/**
+ * Displays a list of available project roles
+ * @param {Project} project object
+ * @returns {ReactElement} a list of projects
```

Comment:
I think this shows the detailed info about one project, rather than a list of projects?

Reply:
ah yup! Getting the screens muddled up 😆